src/pages/develop/projects.md

```diff
@@ -6,7 +6,7 @@ A Railway project contains all of your plugins, environments, and deployments. I
 
 All projects feature
 
-- 100 GB outbound network bandwidth
+- 80 GB outbound network bandwidth
```

Comment:
Don't think this was supposed to be changed

Reply:
That was a mistake on my part. Will fix.